content/en/docs/reference/controllers/certificate-approver.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Certificate signature approver
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The CertificateSigningRequest approver controller is part of a set of built-in
+controllers for certificate management.
```

Comment:
- introduce the acronym at line 8?
- what is the actual name; do you need `approver`?